packages/next/package.json

```diff
@@ -61,6 +61,7 @@
     "get-port": "5.0.0",
     "nanoid": "3.3.4",
     "ndjson": "2.0.0",
+    "pretty-bytes": "^6.0.0",
```

Comment:
Seems like the `save-prefix: ""` setting didn't work because the up-caret is still there.

Perhaps we should use `save-exact: true` instead.

Reply:
save-exact is not supported by pnpm so just adding save-prefix which is

Reply:
Oddly enough, `save-exact=true` work for me with pnpm even though its not documented

Reply:
are you putting that in `.npmrc` ? I tried it and it didn't work for some reason

Reply:
Yes in `.npmrc` with `pnpm@7.22.0`

https://github.com/styfle/links-awakening/blob/main/.npmrc

Oddly enough, I couldn't get `save-prefix` to work when I tried it last.

Reply:
lol really? that is interesting. let me try again

Reply:
okay both seem to work. ill stick with save-exact=true